TODO.md

```diff
@@ -1,9 +1,11 @@
 # TODO
 1. Write a script for setting up Windows/VisualStudio. Currently only needed by Bora. Ask him if you need this.
 2. Hotloading
```

Comment:
Hotloading is also implemented now. Please move it to Completed tasks as well.

Reply:
Done!